README.md

```diff
@@ -1,24 +1,17 @@
 What is this?
 =============
-
 See what this is all about: [http://open-learning-exchange.github.io/](http://open-learning-exchange.github.io/).
```

Comment:
This is actually [broken](https://github.com/SamriddhiBasnet1/SamriddhiBasnet1.github.io/blob/3ecfd7537ef079c11f8c711a2287394600435e58/README.md) after you remove the extra lines.